src/js/index.js

```diff
@@ -117,8 +136,7 @@ const getCards = (cards) => {
 homesSlider();
 
 if (!sessionStorage.getItem('hotels')) {
-  getHomes('https://fe-student-api.herokuapp.com/api/hotels').then((data) => {
-    getCards(data);
+  getHomes(url).then((data) => {
```

Comment:
add also function call getCards in block if, because now  your block with hotels loading only, then data saved in session storage